data/shared/citizen/scripting/v8/console.js

```diff
@@ -270,6 +271,10 @@
             this._trace(format(message, ...optionalParams));
         }
 
+		logSync(message = undefined, ...optionalParams) {
```

Comment:
and after all that nitpicking/diff pollution/whatever it is by removing whitespace, you're using tabs in a file that only has spaces otherwise?!